main.py

```diff
@@ -5031,6 +5026,102 @@ def do_command(self, command, args, sender, channel, tags, isWhisper=False):
                                  isWhisper)
                 return
+            if command == "godimage":
+                canManageImages = sender in superadmins
+                godRarity = int(config["numNormalRarities"]) - 1
+                if len(args) < 1:
+                    if canManageImages:
+                        self.message(channel, "Usage: !godimage change / changeglobal / queue / check / acceptsingle / acceptglobal / reject", isWhisper)
+                    else:
+                        self.message(channel, "Usage: !godimage change / changeglobal / list / cancel", isWhisper)
+                    return
+                subcmd = args[0].lower()
+                if subcmd in ["change", "changeglobal", "request", "requestglobal"]:
+                    do_global = subcmd in ["changeglobal", "requestglobal"]
+                    if len(args) < 3:
+                        self.message(channel, "Usage: !godimage change[global] <id> <link>", isWhisper)
+                        return
+                    try:
+                        waifuid = int(args[1])
+                    except ValueError:
+                        self.message(channel, "Usage: !godimage change[global] <id> <link>", isWhisper)
+                        return
+
+                    with db.cursor() as cur:
+                        cur.execute("SELECT COUNT(*) FROM has_waifu WHERE userid = %s AND waifuid = %s AND rarity = %s", [tags['user-id'], waifuid, godRarity])
+                        if cur.fetchone()[0] == 0:
+                            self.message(channel, "You don't own that waifu at god rarity!", isWhisper)
+                            return
+                        
+                        try:
+                            validateImageURL(args[2])
+                        except ValueError as ex:
+                            self.message(channel, "Invalid link specified. %s" % str(ex), isWhisper)
+                            return
+
+                        if canManageImages:
+                            # automatically do the change
+                            try:
+                                hostedURL = processImageURL(args[2])
+                            except Exception as ex:
+                                self.message(channel, "Could not process image. %s" % str(ex), isWhisper)
+                                return
+                            
+                            if do_global:
+                                cur.execute("UPDATE waifus SET image = %s WHERE id = %s", [hostedURL, waifuid])
+                            else:
+                                cur.execute("UPDATE has_waifu SET custom_image = %s WHERE waifuid = %s AND userid = %s AND rarity = %s", [hostedURL, tags['user-id'], waifuid, godRarity])
+
+                            # log the change for posterity
+                            insertArgs = [tags['user-id'], waifuid, args[2], do_global, tags['user-id'], current_milli_time()]
```

Comment:
You're using `tags['user-id']` for both the `requesterid` and `moderatorid` fields? Is this really what you meant?